app/views/notifications/_notifications_list.html.erb

```diff
@@ -1,17 +1,8 @@
-<% @notifications.each do |notification| %>
+<% render_each_notification_or_error @notifications, error: "notifications/shared/error" do |notification| %>
   <div
     class="mb-2 spec-notification <%= "unseen" unless notification.read? %>"
     data-notification-id="<%= notification.id %>">
-    <%= render notification.notifiable_type.downcase.to_s, notification: notification %>
-  </div>
-
-  <% rescue => e %>
```

Comment:
Just wanted to note the motivation here: `rescue`ing _all_ errors in this way makes debugging very opaque in this area. (Especially when combined with the layers of partials, it took some time to even discover where this was happening.) I figured I could just disable it temporarily, but I also thought there was a good chance I would accidentally commit that change, so tried to find a more robust solution. Long-term, I think we might want a separate environment flag (`debug?` or something?) but I decided that was beyond the scope of this PR (which was supposed to be setting up the new reactions, not really intended as a `notifications` refactor).